internal/learn/contribute-to-wiki/getting_started/developer.md

```diff
@@ -26,22 +22,33 @@ Active technical writers:
 
 See [Setting Up Documentation](../how_tos/documentation_setup.md).
 
-## Do Your Homework
+## Write Documentation
+
+There are [four kinds of documentation](../reference/structure/overview.md). The reference can be generated automatically from inline comments in your code: say, with [rustdoc](https://doc.rust-lang.org/rustdoc/index.html), [typedoc](https://typedoc.org/), [javadoc](https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html), or [pydoc](https://docs.python.org/3/library/pydoc.html). Tutorials are handled by the IOTA Foundation's DevX team. This leaves you with [how to guides](../how_tos/how_to_write_how_tos.md) and explanation articles.
```

Comment:
```suggestion
There are [four kinds of documentation](../reference/structure/overview.md). You can generate [reference](../reference/structure/reference) automatically from inline comments in your code using: 

* [rustdoc](https://doc.rust-lang.org/rustdoc/index.html)
* [typedoc](https://typedoc.org/)
* [javadoc](https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html)
* [pydoc](https://docs.python.org/3/library/pydoc.html). 

[Tutorials](../reference/structure/tutorials) are handled by the IOTA Foundation's DevX team. This leaves you with [how to guides](../how_tos/how_to_write_how_tos.md) and [explanation articles](../reference/structure/explanations).
```

Reply:
I wanted to have something before we have a dedicated article, but I would prefer to keep it as a paragraph for now to keep the flow. We'll replace it with a link to the article afterwards anyway.